### ISSUE TITLE
Aimbot doesnt seem to work at all?

### ISSUE THREAD

**Issue by DacianKing** [ArnoldasMk/Kali#13]
I dont know why but the aimbot doesnt work at all. Some features work like auto-aim, but seems like thats it

**Comment by ArnoldasMk** [comment]
You're talking about legitbot? Please mention all features that don't work for you. Also update if you haven't yet.

**Comment by DacianKing** [comment]
Yes sorry it's working now for some reason, however the aimbot doesnt work properly because of the recoil. How do I toggle on no recoil?

**Comment by DacianKing** [comment]
The aimbot aims far too high above the body to hit properly

**Comment by ArnoldasMk** [comment]
> The aimbot aims far too high above the body to hit properly

Is this with recoil control you're talking about or it's by default ?


> Yes sorry it's working now for some reason, however the aimbot doesnt work properly because of the recoil. How do I toggle on no recoil?

You can toggle it in Legitbot -> Recoil Control -> Always On

**Comment by DacianKing** [comment]
yes with recoil control. it doesnt seem to work at all

**Comment by DacianKing** [comment]
I have to actually aim downwards myself as the "snap" is less powerful than the recoil. So if you go rage hacking and with autoshoot it will miss.

**Comment by ArnoldasMk** [comment]
Okay, I will look at this after I push a big update, thanks for reporting
